frontend/src/metabase/visualizations/visualizations/Table.tsx

```diff
@@ -355,7 +356,6 @@ class Table extends Component<TableProps, TableState> {
           rows: rows.map(row => columnIndexes.map(i => row[i])),
           results_timezone,
         },
-        // cache question for performance reasons
```

Comment:
This is not true atm. There is a comment above "// construct a Question that is in-sync with query results" that is accurate.